### PR TITLE
js-party-60-unintelligibles

### PR DESCRIPTION
**Pull request by BlackAzalea**
fixed unintelligibles for JS Party #60 podcast

### CODE REVIEW

File: jsparty/js-party-60.md [hunk 1]
```diff
@@ -172,9 +172,9 @@ This was a pretty big productivity destroyed for me, given that I've just lost s
...
 
-So I was sort of tearing my hair out, and one of my colleagues came into the room, and we were sort of like going back and forth on what it could be, and they're like "Do you mind if I just jump on your computer?" They were running all of these different commands, and I was like, "Yeah, this is great, to have a second set of eyes on it." And right as he's running a bunch of stuff, I'm like "Oh, what about /etc/hosts? What if I have something in there that's blocking a specific URL to pull that down?" He said, "No, I already \[unintelligible 00:26:46.21\] that out. There's nothing actually in there."
+So I was sort of tearing my hair out, and one of my colleagues came into the room, and we were sort of like going back and forth on what it could be, and they're like "Do you mind if I just jump on your computer?" They were running all of these different commands, and I was like, "Yeah, this is great, to have a second set of eyes on it." And right as he's running a bunch of stuff, I'm like "Oh, what about /etc/hosts? What if I have something in there that's blocking a specific URL to pull that down?" He said, "No, I already cutted that out. There's nothing actually in there."
```
Comment (pnevares) [inline jsparty/js-party-60.md]: In context I think this is `cat`-ed, not cutted.

File: jsparty/js-party-60.md [hunk 2]
```diff
@@ -172,9 +172,9 @@ This was a pretty big productivity destroyed for me, given that I've just lost s
...
+He jumped off the computer, and then I just ran cat/etc/hosts just in case, and sure enough, in July - three months prior to September, when stuff stopped working - I actually remembered that I had agreed to beta test a brand new kind of configuration setting... And because they didn't have a feature flag for it, they said "The quickest thing for you to do is to just add this entry to your /etc/hosts, and when it goes to lazy-load this module, it will pull it down from a different IP address." They had hosted that on a random IP address, somewhere in a VM or something like that, and that's how they were actually doing it. So that was just not fun...
```
Comment (pnevares) [inline jsparty/js-party-60.md]: `cat /etc/hosts` (with the space)

File: jsparty/js-party-60.md [hunk 3]
```diff
@@ -268,7 +268,7 @@ I think in that case, if you do have long periods of time - which is totally not
...
+**Kevin Ball:** Oh yeah, I basically use what Suz described, except I do pretty much all of my development branches, so it's never master that's dirty; it's always some other branch. But yeah, then I'll do a commit and I'll just say 'stash forward progress' and then what I can essentially do is then you do 'get reset head 1' and that gets you back to that exact dirty state that you're talking about. I'll commit it, so that I'm able to go back to a master branch or some other branch if there's a need, but then when I wanna get back to that, "Okay, just show me what it is with a Git diff", and then I can reset it.
```
Comment (pnevares) [inline jsparty/js-party-60.md]: Try `git reset HEAD^1`

**Comment by BlackAzalea** [comment]
Thank you! I made the changes. I have much to learn.

**Pull request merged**